vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/ItemClickListenerIT.java

```diff
@@ -79,6 +88,15 @@ public void clickCell_columnNameAvailable() {
         Assert.assertEquals("Name", getColumnClickMessage());
     }
 
+    @Test
+    public void clickComponentInCell_columnNameAvailable() {
```

Comment:
This was correctly removed in previous commit and now you brought it back. :)

Reply:
You know what, I will get back to this tomorrow. I am afraid this will not go anywhere good today.